cax/cax.json

```diff
@@ -34,7 +34,9 @@
     "directory": "/project/lgrandi/xenon1t",
     "upload_options": [],
     "username": "tunnell",
-    "download_options": []
+    "download_options": [],
+    "pax_versions": ["v4.10.0"],
+    "pax_locations": ["/project/lgrandi/xenon1t/root/pax_v4.10.0"]
```

Comment:
Can this be a list of dicts?  "pax" : {"v4.10.0" : "/project/lgrandi/xenon1t/root/pax_v4.10.0"}